setup.cfg

```diff
@@ -16,7 +16,12 @@ envlist = py36, py37, py38, lint
 minversion = 3.20.1
 
 [testenv:py{36,37,38}]
-commands = python -m unittest -vb {posargs}
+allowlist_externals = sh
+commands_pre =
+  # install nprint into environ via bootstrap & check that it worked
+  sh -c 'test "$(which nprint)" = "{envbindir}/nprint" || nprint-install -s && test "$(which nprint)" = "{envbindir}/nprint"'
```

Comment:
This both performs a sanity test of the bootstrapping command and satisfies the (forthcoming) requirement that nPrint be available to tests.